src/Compilers/CSharp/Portable/Symbols/NamedTypeSymbol.cs

```diff
@@ -577,7 +577,7 @@ private void CalculateRequiredMembersIfRequired()
         private bool TryCalculateRequiredMembers(ref ImmutableSegmentedDictionary<string, Symbol>.Builder? requiredMembersBuilder)
         {
             var lazyRequiredMembers = _lazyRequiredMembers;
-            if (_lazyRequiredMembers == RequiredMembersErrorSentinel)
+            if (lazyRequiredMembers == RequiredMembersErrorSentinel)
             {
                 if (lazyRequiredMembers.IsDefault)
```

Comment:
I didn't understand under what conditions we can:
1. have 'lazyRequiredMembers' be equal to the error sentinel (which is non-empty, looks like?)
2. and then have it turn out to be default.

Could you please clarify?